src/apps/something-similar/SomethingSimilar.entry.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from "react";
+import SomethingSimilar from "./SomethingSimilar";
+import { withText } from "../../core/utils/text";
+import { withConfig } from "../../core/utils/config";
+import { withUrls } from "../../core/utils/url";
+
+export interface SomethingSimilarProps {
+  fbsBaseUrl: string;
+  publizonBaseUrl: string;
+  dplCmsBaseUrl: string;
+  coverBaseUrl: string;
+  materialBaseUrl: string;
+  fbiBaseUrl: string;
+  materialUrl: string;
+  somethingSimilarTitleText: string;
+  materialByAuthorText: string;
+  materialAndAuthorText: string;
+  somethingSimilarByTheSameAuthorText: string;
+  somethingSimilarSomethingSimilarAuthorText: string;
+}
+
+const SomethingSimilarEntry: FC<SomethingSimilarProps> = () => (
+  <SomethingSimilar />
```

Comment:
I think you should use wid as a prop here instead of trying to get it from the url later on. It brings the flexibility of being able to use the app in other contexts as well.
We did the same in the [work/material app](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/f6b21e6ebf7ccc0eab3e0ba03326b32d06a8660b/src/apps/material/material.entry.tsx#L152-L156)

Reply:
done